tests/Resources/ti.ui.navigationwindow.addontest.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2017-Present by Axway. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe.windowsMissing('Titanium.UI.NavigationWindow', function () {
+	var nav;
+
+	this.timeout(10000);
+
+	afterEach(function () {
+		if (nav) {
+			nav.close();
+		}
+		nav = null;
+	});
+
+	it('open window from open event of window (TIMOB-26838)', function (finish) {
+		var window = Ti.UI.createWindow(),
+			navigation = Ti.UI.createNavigationWindow({
+				window: window
+			});
+
+		var nextWindow = Ti.UI.createWindow();
+
+		nextWindow.addEventListener('open', function () {
+			finish();
+		});
+		window.addEventListener('open', function () {
+			nav.openWindow(win2,{animated:true});
```

Comment:
<!--
  5 failure:  tests/Resources/t..., tests/Resources/t..., tests/Resources/t..., tests/Resources/t..., tests/Resources/t...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: tests/Resources/ti.ui.navigationwindow.addontest.js;
  Line: 37;
-->
- :no_entry_sign: tests/Resources/ti.ui.navigationwindow.addontest.js line 37 – 'win2' is not defined. (no-undef)
- :no_entry_sign: tests/Resources/ti.ui.navigationwindow.addontest.js line 37 – A space is required after ','. (comma-spacing)
- :no_entry_sign: tests/Resources/ti.ui.navigationwindow.addontest.js line 37 – A space is required after '{'. (object-curly-spacing)
- :no_entry_sign: tests/Resources/ti.ui.navigationwindow.addontest.js line 37 – A space is required before '}'. (object-curly-spacing)
- :no_entry_sign: tests/Resources/ti.ui.navigationwindow.addontest.js line 37 – Missing space before value for key 'animated'. (key-spacing)